jetson/power_logging/model/benchmark.py

```diff
@@ -149,18 +160,24 @@ def benchmark(args: argparse.Namespace) -> None:
         if (save_dir / "val").exists():
             shutil.rmtree(save_dir / "val")
 
+        ## Used for ONNX cpu time
```

Comment:
Is it still needed? Can we make it conditional easily? (If not trivially easy, I wouldn't bother right now)

Reply:
It's bit tricky and one-off situation I think, both pytorch and tensorrt use GPU. If we could get ONNX working on GPU, this is not required at all.